app/metal-controller-manager/internal/ipxe/ipxe_server.go

```diff
@@ -50,7 +52,7 @@ exit
 
 var (
 	apiEndpoint          string
-	extraAgentKernelArgs []string
```

Comment:
change format to whatever kernel args format is (with spaces), as commas have special meaning in kernel args